pages/signup.php

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="signup.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500;600;700;900&display=swap" rel="stylesheet">
+    <title>SignUp Page</title>
+</head>
+<body>
+    <div class="container">
+        <div class="component-1">
+            <img id="logo" src="../assets/img/logo.svg" alt="logo" width="182px" height="40px" >
+           
+            <h2>Get Started!</h2>
+            <h4>Please sign up to continue</h4>
+
+            <div class="form">
+                <label for="">Email</label>
+                <input type="email" name="" id="" placeholder="mail@example.com">
+
+                <label for="">User Name</label>
+                <input type="text" name="" id="" placeholder="madhush99">
+
+                <label for="">Password</label>
+                <input type="password" name="" id="" placeholder="********">
+
+                <label for="">Confirm Password</label>
+                <input type="password" name="" id="" placeholder="********">
+            </div>
+
+            <div class="position">
+                <input type="radio" name="position" id="">
+                <label for="Student">Student</label>
+                <input type="radio" name="position" id="">
+                <label for="Student">Admin</label>
+            </div>
+            
+            <div class="terms-condition">
+                <input type="checkbox" name="" id="">
```

Comment:
Please provide a value for every `name` attribute that is inside the form tag input elements.
- id issue